tiles/src/main/java/com/protomaps/basemap/layers/Pois.java

```diff
@@ -278,6 +278,11 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
           } else {
             minZoom = 15;
           }
+
+          // Hack for weird San Francisco university
```

Comment:
This is a hack, but the map in SF looks really bad without it. OK to keep?

Reply:
Probably we want to scope down more so it doesn't test every feature's name...

Reply:
With the fix:

<img width="1053" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/33206c01-5749-4b29-a398-42408c186bbe">

Without the fix (hack) the weird SF university is shown at zoom 11 – even though no other POIs are showing. (Big parks outside the city show at this zoom, as well as airports.)

<img width="1108" alt="image" src="https://github.com/protomaps/basemaps/assets/853051/37193a04-c33b-4299-8403-2d145b2e790c">

Reply:
Is this an urban/rural divide issue? Like it looks strange in context of it being in a city, but at the same zoom level and same prominence in Davis it would look OK/

Reply:
Maybe? UC Davis has a large, regularly shaped polygon. This urban college is something else entirely. Perhaps more like NYU but in many different buildings 20+ blocks away from each mini campus versus having clusters of entire city blocks.